conf/resources/data/sa/transitionedToiTSAMigratedWithNoDebtAndLiability.json

```diff
@@ -115,8 +108,17 @@
     ],
     "signals": [
       {
+        "signalType": "Transitioned to CDCS",
+        "signalValue": "True",
+        "signalDescription": "ETMP Transitioned to CDCS signal type"
+      },
+      {
+        "signalType": "ITSA MigraTed",
+        "signalValue": "False",
```

Comment:
ITSA Migrated signals have dates, not booleans. Its only the CDCS signals that have dates